MohamedH1998/99NamesAPI
# Implement logging solution using winstonjs

## What has changed
1. Implemented a `logger` by using winston to create a new logger.
2. Fixed some tests that were breaking due to bugs in the API code.

## How to test
1. Run `npm run dev` and make sure that the logger logs outputs for each API route
2. Run `npm run test` and make sure all tests are passing.
